packages/es-dev-server/demo/plugin-serve/README.md

```diff
@@ -0,0 +1 @@
+erewrewrweewrw
```

Comment:
you wanna keep this file? maybe add some context what it's for? 😅

Reply:
uhhh nope we don't :D